dashboard/cron.py

```diff
@@ -284,4 +282,4 @@ def util_function(sql_string, mysql_table, table_identifier=None):
     df.drop_duplicates(keep=False, inplace=True)
 
     # write to MySQL
-    df.to_sql(con=engine, name=mysql_table, if_exists='append', index=False)
\ No newline at end of file
+    df.to_sql(con=engine, name=mysql_table, if_exists='append', index=False)
```

Comment:
There is no change made to this line, right?

Reply:
Doesn't look like it but that reminds me I need to readd psycopg2 as you were using it. Also I need to either get this image or a fork of this image into the openshift containers before committing this so this is still WIP. I think I'll fork this project so we're not reliant on another users project. The big reason I'm using this instead of a more official image was because it has to build pandas from source which is slow, we can pre-compile all of that in an image and put that up!